lib/pure/strutils.nim

```diff
@@ -89,6 +89,9 @@ const
     ## All the characters that count as whitespace (space, tab, vertical tab,
     ## carriage return, new line, form feed).
 
+  Punctuation* = {'\33'..'\47', '\58'..'\64', '\91'..'\96', '\123'..'\126'}
```

Comment:
why obfuscate with ord values, can't you instead use ',' etc?

Reply:
Changed to symbols.